src/by/epam/ayem/module4/service/CountryService.java

```diff
@@ -0,0 +1,86 @@
+package by.epam.ayem.module4.service;
+
+/*3. Создать объект класса Государство, используя классы Область, Район, Город. Методы:
+вывести на консоль столицу, количество областей, площадь, областные центры.*/
+
+import by.epam.ayem.module4.model.City;
+import by.epam.ayem.module4.model.Country;
+import by.epam.ayem.module4.model.District;
+import by.epam.ayem.module4.model.Region;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class CountryService {
+
+    private Set<Country> countries;
+    private Set<Region> regions;
+    private Set<District> districts;
+    private List<City> cities;
+
+    public CountryService() {
+        this.countries = new HashSet<>();
+        this.regions = new HashSet<>();
+        this.districts = new HashSet<>();
+        this.cities = new ArrayList<>();
+    }
+
+    public void addCity(City city) {
+        cities.add(city);
+    }
+
+    public void addDistrict(District district) {
+        districts.add(district);
+    }
+
+    public void addRegion(Region region) {
+        regions.add(region);
+    }
+
+    public void addCountry(Country country) {
+        countries.add(country);
+    }
+
+    public void countArea(Country country) {
```

Comment:
All method names should start from verb. So I would rename it to `showCountryArea`